docs/api/proxy-based-tracking.md

```diff
@@ -0,0 +1,145 @@
+---
+id: proxy-based-tracking
+title: Proxy-based Tracking
+sidebar_label: Proxy-based Tracking
+hide_title: true
+---
+
+# Proxy-based Tracking
+
+This document describes about `useTrackedState` hook.
+
+## How does this get used?
+
+`useTrackedState` is a hook that can be used instead of `useSelector`.
+It doesn't mean to replace `useSelector` completely.
+It gives a new way of connecting Redux store to React.
+
+> **Note**: It's not completely new in the sense that there already exists a library for `connect`: [beautiful-react-redux](https://github.com/theKashey/beautiful-react-redux)
+
+The usage of `useTrackedState` is extremely simple.
```

Comment:
I'd like to avoid using phrases like "simple" and "intuitive" in the docs if possible.

Reply:
Makes sense.